lib/errors.js

```diff
@@ -0,0 +1,57 @@
+'use strict';
+
+const errorTitle = (code) => `${code}_ERROR`;
+const errorMsg = (code) => `There is a ${code}_ERROR`;
+
+const getContent = (err, debug, translate) => {
+
+  const content = {};
+
+  if (err.code === 'SESSION_TIMEOUT') {
+    content.title = (translate && translate('errors.session.title'));
+    content.message = (translate && translate('errors.session.message'));
+  }
+
+  if (err.code === 'NO_COOKIES') {
+    err.status = 403;
+    content.title = (translate && translate('errors.cookies-required.title'));
+    content.message = (translate && translate('errors.cookies-required.message'));
+  }
+
+  err.code = err.code || 'UNKNOWN';
+
+  if (!content.title) {
+    content.title = (translate && translate('errors.default.title')) || errorTitle(err.code);
+  }
+
+  if (!content.message) {
+    content.message = (translate && translate('errors.default.message')) || errorMsg(err.code);
+  }
+
+  return content;
+};
+
+module.exports = (options) => {
+
+  const logger = (options || {}).logger;
+  const translate = (options || {}).translate;
+  const debug = (options || {}).debug;
```

Comment:
What would be neater would be to use default arguments and object destructuring, when node catches up :)

```js
module.exports = (options = {}) {
  const { logger, translate, debug } = options;
}
```